corehq/util/es/elasticsearch.py

```diff
@@ -32,8 +32,27 @@
         SnapshotClient,
     )
     from elasticsearch5.helpers import BulkIndexError, bulk
+elif settings.ELASTICSEARCH_MAJOR_VERSION == 6:
+    import elasticsearch6 as elasticsearch
+    from elasticsearch6.exceptions import AuthorizationException
+    from elasticsearch6 import (
+        ConnectionError,
+        ConflictError,
+        ConnectionTimeout,
+        Elasticsearch,
+        ElasticsearchException,
+        NotFoundError,
+        SerializationError,
+        TransportError,
+        RequestError,
+    )
+    from elasticsearch5.client import (
+        IndicesClient,
+        SnapshotClient,
+    )
+    from elasticsearch5.helpers import BulkIndexError, bulk
```

Comment:
Are these imports an indication that we need to keep `elasticsearch5` (which I assume is a library) around even after we have upgraded to ES6?

Reply:
The purpose that they serve is that we can support multiple versions simultaneously when we are in process of upgrading. After that it is totally fine to remove the older version and just keep the one that is being used.